types/discountCodes.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+/* Discount Codes */
+export type CodeData = {
+  name?: Object;
+  description?: Object;
+  cartDiscounts: [];
+  cartPredicate?: string;
+  isActive: boolean;
+  maxApplicationsPerCustomer?: number;
```

Comment:
This is called `maxApplications`

http://dev.commercetools.com/http-api-projects-discountCodes.html#discountcode